docs/lib/graphqlapi/fragments/flutter/advanced-workflows/10_example.md

```diff
@@ -0,0 +1,28 @@
+```dart
+try{
+    String graphQLDocument = '''mutation createTodo(\$input: CreateTodoInput!) {
+      createTodo(input: \$input) {
+        id
+        name
+        description
+      }
+    }''';
+
+    var operation = await Amplify.API.mutate(
+        request: GraphQLRequest<String>(
+            document: graphQLDocument,
+            variables: {
+              "name": "my first todo",
+              "description": "todo description"
+            }
+        )
+    );
+
+    var response = await operation.response;
+    var data = response.data;
+    
+    print("Mutate SUCCESS");
```

Comment:
If you access the data, why not print it? Can this be just `print("Mutate succeeded: " + operation.response.data);`?

Reply:
Sure, though depending on that data stored not too sure what will be returned.  Typically it's just going to be byte data so it's a bit of gibberish just to print it out like that. 

We could instead add something like this: 
      print(new String.fromCharCodes(response.data));

Which, if the data stored was a string beforehand, would display that data again as a string.  However, I thought that arbitrary data could be stored via these methods, so this might be hard coding an example for a specific case @jamesonwilliams ?

Reply:
offline talk, should add the following:

```
// For String JSON response 
print(new String.fromCharCodes(response.data));

```